pipe.go

```diff
@@ -159,6 +184,10 @@ func DialPipe(path string, timeout *time.Duration) (net.Conn, error) {
 		}
 		err = waitNamedPipe(path, ms)
 		if err != nil {
+			if shouldRetry(err) {
+				time.Sleep(time.Duration(rand.Intn(10)) * time.Millisecond)
+				continue
```

Comment:
If we continue here, do we have to close `h` to avoid a handle leak?

Reply:
Unless I misunderstood, `CreateFile()` only allocates a handle when it succeeds and we never reach this path if it does:

```go
for {
    h, err = createFile(path, ...)
    if err == nil {
        break
    }
    ...
    err = waitNamedPipe(path, ...)
}
```

Reply:
Correct. Now I see it (on a bigger screen than my mobile phone, haha :-) ).